e3sm_diags/derivations/derivations.py

```diff
@@ -739,9 +874,14 @@
         ("surf_radiative_T",): rename,  # EAMxx
     },
     "PS": {("ps",): rename},
-    "U10": {("sfcWind",): rename},
+    "U10": {
+        ("sfcWind",): rename,
+        ("wind_speed_10m",): rename,  # EAMxx ?
```

Comment:
Is there supposed to be a `?` in the comment here?

Reply:
@crterai could you confirm is `wind_speed_10m` is equivalent to U10 from EAM?

Reply:
Yes, `wind_speed_10m` is the 10m wind speed, which I'm guessing `U10` to represent